### PR TITLE
Hide the buttons to see hierarchy and to manage hierarchy of institution

### PR DESCRIPTION
**Pull request by mayzabeel**
<p><b>Feature/Bug description:</b>The feature to send hierarchical invitations will be not available on first version </p>

<p><b>Solution:</b> Hide the buttons that see and manage the hierarchical invitations and throw exception when the methods ate called directly by route. The backend tests are commented. </p>

<p><b>TODO/FIXME:</b>Undo when the feature can be avaiable</p>

screenshot from 2018-01-10 16-14-24

screenshot from 2018-01-10 16-14-11

screenshot from 2018-01-10 16-13-55

### CODE REVIEW

File: frontend/institution/base_institution_page.html [hunk 1]
```diff
@@ -83,7 +83,9 @@
                                             Seguidores
                                         </md-button>
                                     </md-menu-item>
-                                    <md-menu-item>
+                                    <!--TODO: Show the item below when hierarchical invitations can be sent 
+                                    @author: Mayza Nunes 10/01/17-->
+                                    <md-menu-item ng-if="false">
```
Comment (andreldsa) [inline frontend/institution/base_institution_page.html]: To improve security, we should protect the handlers from direct access. An option it creates an exception that throws an error when a function it's disabled.

Comment (mayzabeel) [reply frontend/institution/base_institution_page.html]: The handler who send invite for hierarchy institution is the same used to send invite for institutions without hierarchy, so it's not possible throws an error when the post is called.

File: backend/handlers/institution_children_request_collection_handler.py [hunk 2]
```diff
@@ -44,6 +45,12 @@ def post(self, user, institution_key):
 
         type_of_invite = data.get('type_of_invite')
 
+        """TODO: Remove the assert bellow when the hierarchical requests can be avaiable
+        @author: Mayza Nunes 11/01/2018
+        """
+        Utils._assert(type_of_invite == 'REQUEST_INSTITUTION_CHILDREN',
+                      "Hierarchical requests is not avaiable on test version", NotAuthorizedException)
```
Comment (andreldsa) [inline backend/handlers/institution_children_request_collection_handler.py]: "*available in this version."

File: backend/handlers/institution_parent_request_collection_handler.py [hunk 3]
```diff
@@ -44,6 +45,12 @@ def post(self, user, institution_key):
 
         type_of_invite = data.get('type_of_invite')
 
+        """TODO: Remove the assert bellow when the hierarchical requests can be avaiable
+        @author: Mayza Nunes 11/01/2018
+        """
+        Utils._assert(type_of_invite == 'REQUEST_INSTITUTION_PARENT',
+                      "Hierarchical requests is not avaiable on test version", NotAuthorizedException)
```
Comment (andreldsa) [inline backend/handlers/institution_parent_request_collection_handler.py]: *in this version.

File: backend/handlers/invite_collection_handler.py [hunk 4]
```diff
@@ -35,8 +35,16 @@ def post(self, user):
...
         type_of_invite = data.get('type_of_invite')
+
         Utils._assert(type_of_invite == 'INSTITUTION',
                       "invitation type not allowed", NotAuthorizedException)
+        
+        """TODO: Remove the assert bellow when the hierarchical invitations can be avaiable
+        @author: Mayza Nunes 11/01/2018
+        """
+        Utils._assert(type_of_invite != 'USER',
+                      "Hierarchical invitations is not avaiable on test version", NotAuthorizedException)
```
Comment (andreldsa) [inline backend/handlers/invite_collection_handler.py]: ...

File: backend/test/institution_children_collection_request_handler_test.py [hunk 5]
```diff
@@ -56,8 +56,23 @@ def test_post(self, verify_token):
...
 
-        request = self.testapp.post_json("/api/institutions/" + institution.key.urlsafe() + "/requests/institution_children", data)
+        with self.assertRaises(Exception) as ex:
+            self.testapp.post_json(
+                "/api/institutions/" + institution.key.urlsafe() + "/requests/institution_children",
+                data)
 
+        exception_message = self.get_message_exception(ex.exception.message)
+        self.assertEqual(
+            "Error! Hierarchical requests is not avaiable on test version",
```
Comment (andreldsa) [inline backend/test/institution_children_collection_request_handler_test.py]: *in this version.

File: frontend/institution/management_members.html [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <!-- CONTENT -->
-<div flex-gt-lg="80" flex-lg="90" layout-align="center" layout-fill>
+<div layout-padding flex-gt-sm="80" flex-gt-lg="80" flex-lg="90" layout-align="center" layout-fill>
```
Comment (andreldsa) [inline frontend/institution/management_members.html]: The property flex it's not correct.  It's missing the flex default, and the tag "-gt-" it's only used in the last definition of the div:

`<div flex flex-sm="80" flex-md="80" flex-lg="70" flex-gt-lg="60" ...>`

**Pull request merged**